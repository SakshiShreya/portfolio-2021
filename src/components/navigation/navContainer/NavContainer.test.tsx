import { HashRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import NavContainer from "./NavContainer";

it("should render NavContainer properly", () => {
  const elem = renderer
    .create(
      <HashRouter>
        <NavContainer />
      </HashRouter>
    )
    .toJSON();
  expect(elem).toMatchSnapshot();
});
