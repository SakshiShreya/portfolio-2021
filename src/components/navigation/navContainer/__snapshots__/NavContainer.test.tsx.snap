// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NavContainer properly 1`] = `
<div
  className="cont"
>
  <nav
    className="drawer "
  >
    <img
      alt="Logo"
      className="logo"
      src="/assets/logo/SSlogo.png"
    />
    <ul>
      <li
        className="li h4"
      >
        <a
          href="#/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="li h4"
      >
        <a
          href="#/about"
          onClick={[Function]}
        >
          About Me
        </a>
      </li>
      <li
        className="li h4"
      >
        <a
          href="#/experience"
          onClick={[Function]}
        >
          Experience
        </a>
      </li>
      <li
        className="li h4"
      >
        <a
          href="#/contact"
          onClick={[Function]}
        >
          Contact me
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="cont slide"
    id="content"
  >
    <div
      className="cont"
    >
      <div>
        <p
          className="h1 head"
        >
          Hello World... 
        </p>
        <h2
          className="h1 head name"
        >
          I am 
          <span
            className="green"
            data-testid="green"
          >
            S
          </span>
          akshi
           
          <span
            className="orange"
            data-testid="orange"
          >
            S
          </span>
          hreya
          ,
        </h2>
        <p
          className="h1 head"
        >
          Web Developer
        </p>
        <p
          className="h2"
        >
          and Mentor.
        </p>
        <p
          className="psText"
        >
          I can develop responsive and pixel-perfect websites. I love what I do. I can mentor you to be the same.
        </p>
        <a
          className="btn secondary small border help"
          href="#/contact"
          onClick={[Function]}
        >
          How can I help you?
        </a>
      </div>
    </div>
  </div>
</div>
`;
